.gitignore

```diff
@@ -114,3 +114,15 @@ docs/_book
 test/
 
 # End of https://www.gitignore.io/api/go,vue,node
+
+# Ignore osx DS_store files
+.DS_Store
+
+# Ignore .gitignore
+.gitignore
+
+# Ignore docker-compose.yml
+docker-compose.yml
+
+# Ignore package-lock
+package-lock.json
```

Comment:
I also think we should not ignore docker-compose.yml and package-lock.json files. where do these ignores come from?